### DISCUSSION TITLE
How do I update global configuration parameters？

### DISCUSSION THREAD

**Discussion by congratulation** [orientechnologies/orientdb#9968] [Q&A]
i want to update the value of parameter,but the command 'config set ' looks like the scope is not global ,how can i update?

**Comment by GustavoMD2** [comment]
Hi,

You can edit the config/orientdb-server-config.xml with the follow entry inside **properties** tag:

<entry name="#ParameterName#" value="#ValueOfParameter#"/>

Regards.
